config/locales/en/activemodel.yml

```
@@ -5,6 +5,11 @@ en:
         record_invalid: "Validation failed: %{errors}"
         between: "must be between %{min} and %{max}"
       models:
+        placements/add_placement/steps/mentors:
+          attributes:
+            mentor_ids:
+              blank: Select a mentor or not yet known
+              invalid: Select a mentor or not yet known
```

Is `invalid` now redundant? I think we removed that error type in the end.

```suggestion
              blank: Select a mentor or not yet known
```